Mathlib/Data/Nat/Fib/Basic.lean

```diff
@@ -323,4 +323,24 @@ theorem fib_succ_eq_succ_sum (n : ℕ) : fib (n + 1) = (∑ k in Finset.range n,
       _ = (∑ k in Finset.range (n + 1), fib k) + 1 := by simp [Finset.range_add_one]
 #align nat.fib_succ_eq_succ_sum Nat.fib_succ_eq_succ_sum
 
-end Nat
+theorem five_dvd_fib_five_mul (n : ℕ) : 5 ∣ fib (5 * n) := by
```

Comment:
Already in mathlib as a special case of `Nat.fib_dvd` (note that `fib 5 = 5`).

Reply:
Oh! Yeah, you are right. I wanted to prove the more general result from https://en.wikipedia.org/wiki/Fibonacci_sequence#Primes_and_divisibility.

Reply:
Since the most general result (`Nat.fib_gcd`) is already there you should remove your changes to `Data.Nat.Fib.Basic` entirely.

Reply:
Since the most general result (`Nat.fib_gcd`) is already there you should remove your changes to `Data.Nat.Fib.Basic` entirely.

Reply:
Done